Front-end/expense-management-frontend/src/App.js

```diff
@@ -17,7 +18,8 @@ function App() {
           <Route path="/signup" element={<Signup />} />
           <Route path="/expenselist" element={<ExpenseList />} />
           <Route path="/add-expense" element={<AddExpense />} />
-          <Route path="/" element={<Login />} />  
+          <Route path="/graphical-dashboard" element={<GraphicalDashboard />} />
+          <Route path="/" element={<Login/>} />  
```

Comment:
Always add new code at the end. This will not create any conflicts while merging to main branch

Reply:
Resolved the comment.